File: shanti_education/public/js/stock_entry.js
frappe.ui.form.on("Stock Entry", {
    cost_center: function(frm) {
        var cost_center_doc = frm.doc.cost_center;
        frm.doc.items.forEach(function (d){
			d.cost_center = cost_center_doc
		})
    },
    update_qty_availability: function(frm) {
        var total_outgoing_qty = 0.0;
        var total_incoming_qty = 0.0;
        for (var i = 0; i < frm.doc.items.length; i++) {
            var item = frm.doc.items[i];
            if (item.is_finished_item == 0) {
                total_outgoing_qty += item.qty;
            }
            if (item.is_finished_item == 1) {
                total_incoming_qty += item.qty;
            }
        }
        frm.set_value('total_outgoing_qty', total_outgoing_qty);
        frm.set_value('total_incoming_qty', total_incoming_qty);
    }
});

File: shanti_education/shanti_education/doc_event/stock_entry.py
from frappe import _
import frappe

def on_update(self, method):
	out_and_in_qty(self)
     
@frappe.whitelist()
def out_and_in_qty(self):
    # ongoing qty total
    total_outgoing_qty = 0.0
    total_incoming_qty = 0.0
    for item in self.items:
        if  item.is_finished_item == 0:
            total_outgoing_qty += item.qty
    self.total_outgoing_qty = total_outgoing_qty
    # incoming qty total
    for item in self.items:
        if  item.is_finished_item == 1:
            total_incoming_qty += item.qty
    self.total_incoming_qty = total_incoming_qty